pipelines/vars/forklift_foreman.yml

```diff
@@ -12,7 +12,5 @@ server_box:
         - "fb-verify-packages.bats"
         - "fb-test-foreman.bats"
         - "fb-test-puppet.bats"
-      bats_teardown:
-        - "fb-finish.bats"
```

Comment:
This means it'll now use the default, which is `fb-destroy-organization.bats`. That doesn't work on Foreman itself. I think you need to keep it in but set it to an empty array instead. Same below.